testing/scenario_app/compile_android_aot.sh

```diff
@@ -62,15 +62,19 @@ mkdir -p "$OUTDIR"
 echo "Compiling kernel..."
 
 "$HOST_TOOLS/dart" \
+  --disable-dart-dev \
   "$HOST_TOOLS/gen/frontend_server.dart.snapshot" \
   --sdk-root "$HOST_TOOLS/flutter_patched_sdk" \
-  --aot --tfa --target=flutter \
+  --target=flutter \
+  --no-print-incremental-dependencies \
```

Comment:
what does this do?

Reply:
This makes it skip printingout a long list of dependencies it resolves during compilation, which aren't really useful to us. Not essential for this PR, but nicer to read the output from.